src/messaging/messaging.ts

```diff
@@ -250,6 +251,124 @@ export class Messaging {
       });
   }
 
+  /**
+  * Sends each message in the given array via Firebase Cloud Messaging.
+  *
+  * Unlike {@link Messaging.sendAll}, this method makes a single RPC call for each message
+  * in the given array.
+  *
+  * The responses list obtained from the return value corresponds to the order of `messages`.
+  * An error from this method or a `BatchResponse` with all failures indicates a total failure
+  * -- i.e. none of the messages in the list could be sent. Partial failures or no failures
```

Comment:
We avoid i. e., so for this I'd suggest just "...indicates a total failure, mean that none of the messages in the list could be sent.